public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -57,6 +57,7 @@ type Props = OwnProps & ConnectedProps<typeof connector>;
 class DashNav extends PureComponent<Props> {
   constructor(props: Props) {
     super(props);
+    this.props.dashboard.meta.canShare = false;
```

Comment:
Does this change prevent exporting dashboards, too? It's in that same modal with sharing.

Reply:
Done